composer.json

```diff
@@ -31,7 +31,8 @@
     "typo3/cms-backend": "*",
     "typo3/cms-extbase": "*",
     "typo3/cms-reports": "*",
-    "typo3/cms-filemetadata": "*"
+    "typo3/cms-filemetadata": "*",
+    "solarium/solarium": ">=6.0 <7.0"
```

Comment:
As far as I know Solarium is only required if SolrCell is used, so our suggestions about EXT:solr for SolrCell should be sufficient for "normal" usage and for the unit tests we could add EXT:solr as dev dependency.

Btw. by checking the Solarium dependency I recognized that our Tika Server backend module is only available with EXT:solr, which is not clearly documented, perhaps we should improve this in a separate step.

Reply:
The class is in use directly in https://github.com/TYPO3-Solr/ext-tika/blob/7281b68bc04f27e3e656c1991a133d55f5465f8b/Classes/Report/TikaStatus.php#L196

Reply:
Right, the status check uses Solarium if the SolrCell is configured.

So I still think there is only a dependency to EXT:solr and Solarium if SolrCell is in use, so that we can keep EXT:solr as a suggestion only

Reply:
Solarium is now not a dependency anymore.